src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/topology/AbstractTopologyConfig.java

```diff
@@ -38,6 +38,9 @@
     @Key("parallelism.new")
     int getNewParallelism();
 
+    @Key("isl.latency.parallelism")
+    int getIslLatencyParallelism();
```

Comment:
What about the default value? Can topology code start correctly if it will receive 0 here?

Reply:
Value will always has some value because we are getting value of this param from config

```{{ getv "/kilda_storm_isl_latency_parallelism" }}```

without value in the config Kilda wouldn't start